dev_tools/conf/pip-list-dev-tools.txt

```diff
@@ -24,6 +24,7 @@ qiskit~=0.20.0
 pypandoc
 myst-parser
 Sphinx~=3.2.0
+Pygments==2.6.1
```

Comment:
Optional: I'd add a reference to the relevant issue to aid our future selves in unpinning once the underlying problem is fixed.

Reply:
Thanks, added! Also added an issue to RTD as they weren't aware of this seemingly...:D